pinot-spi/src/main/java/org/apache/pinot/spi/stream/MessageBatch.java

```diff
@@ -116,4 +116,12 @@ default StreamPartitionMsgOffset getOffsetOfNextBatch() {
   default boolean isEndOfPartitionGroup() {
     return false;
   }
+
+  /**
+   * We need this to determine ingestion delay when we receive only null messages (Tombstone messages)
+   * @return last metadata for a null message received by the string
```

Comment:
What is the string here?

Reply:
Yeap! this was expected to be stream :-)